mkosi/resources/mkosi.md

```diff
@@ -125,6 +127,12 @@ The following command line verbs are known:
   OS images, for example via `machinectl pull-raw …` and `machinectl
   pull-tar …`.
 
+`burn <device>`
+
+: This builds the image if it is not built yet, and then writes it to the
+  specified block device. The partition contents is written as-is, but the GPT
```

Comment:
Either 
```suggestion
  specified block device. The partition content is written as-is, but the GPT
```
or
```suggestion
  specified block device. The partition contents are written as-is, but the GPT
```
My preference is on the latter.

Reply:
changed as requested